zerver/openapi/zulip.yaml

```diff
@@ -18191,16 +18223,21 @@ components:
         type: string
       example: "#(?P<id>[0-9]+)"
       required: true
-    LinkifierURLFormatString:
-      name: url_format_string
+    LinkifierURLTemplate:
+      name: url_template
       in: query
       description: |
-        The URL used for the link. If you used named groups for the `pattern`,
+        The [RFC
+        6570](https://www.rfc-editor.org/rfc/rfc6570.html) compliant URL template
+        used for the link. If you used named groups for the `pattern`,
         you can insert their content here with
-        `%(name_of_the_capturing_group)s`.
+        `{name_of_the_capturing_group}`.
+
+        **Changes**: New in Zulip 7.0 (feature level 175). This replaced
+        `LinkifierURLFormatString`.
```

Comment:
Is `LinkifierURLFormatString` a term that's meaningful to the reader of the API docs? I believe it's a name that's purely internal to the implementation of the API docs.

(And might have a role somewhere if generating API bindings from these OpenAPI definitions, too — though since we're not exercising that, there's no guarantee that it makes much sense in that context.)

If I'm understanding correctly the context of how this is used — which is as a parameter in the two routes to create or update a linkifier — then one version would be:

```suggestion
        **Changes**: New in Zulip 7.0 (feature level 175). This replaced
        the parameter `url_format_string`, which was a format string in which
        named groups' content could be inserted with `%(name_of_the_capturing_group)s`.
```